mopidy_tidal/library.py

```diff
@@ -69,69 +62,65 @@ def _get_image_uri(cls, obj):
         cls._log_image_not_found(obj)
 
     def _get_api_getter(self, item_type: str):
-        tidal_lt_0_7_getter_name = f'get_{item_type}'
+        tidal_lt_0_7_getter_name = f"get_{item_type}"
+        # TODO: can't we use nested getattr here?
         return (
             # tidalapi < 0.7.0
             getattr(self._session, tidal_lt_0_7_getter_name)
             if hasattr(self._session, tidal_lt_0_7_getter_name)
             # tidalapi >= 0.7.0
-            else getattr(self._session, item_type)
+            else getattr(self._session, item_type, None)
```

Comment:
Adding a default getattr default return "None" is probably a good idea, although we should probably do it for all getattr() calls to be consistent.

Reply:
I thought this was needed for one code path. But removing it makes no difference to coverage so I clearly got that wrong.  Removed for now.

Reply:
Right, I found what this was supposed to cover.  The code path is probably duplicate.  @BlackLight does tidalapi have any plausible getters set as `None`?  I couldn't find any, but perhaps from when 0.7 was in dev?  Or did you mean to have a default `None` here, as I've modified it to do (in which case I think we can drop this as it's handled in the caller catching `AttributeError`).